### DISCUSSION TITLE
different users menu

### DISCUSSION THREAD

**Discussion by vrs804** [nwg-piotr/nwg-shell#458] [Q&A]
Hello. Is it possible to create another user and have the option in the welcome menu? Is it possible to enter as root option?
I come from arch Endeavour and there are things that are not the same... Great project and good iso to install.

**Comment by nwg-piotr** [comment]
You mean the greeter (nwg-hello)? It should display all existing users. Check the arch wiki for info on how to add users. https://wiki.archlinux.org/title/Users_and_groups

**Reply by vrs804** [reply]
is right .. ( useradd -m user), and ( passwd user ) is all that is necessary.. thanks
